### ISSUE TITLE
Initializing Hive Again Fails

### ISSUE THREAD

**Issue by rayliverified** [IO-Design-Team/hive_ce#26]
**Steps to Reproduce**
Initializing Hive after Hive is already initialize fails. The original behavior in Hive when initializing again is that it returns, without error.

**Code sample**
```
// Press twice
ElevatedButton(
                onPressed: () async {
                  await Hive.initFlutter('storage');
                },
                child: Text('Init Hive'),
              ),

```

**Error:**
```
There is already a TypeAdapter for typeId 200.
packages/hive_ce/src/registry/type_registry_impl.dart 141:11                 registerAdapter
packages/hive_ce_flutter/src/hive_extensions.dart 28:5                       HiveX$124initFlutter
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 84:54           runBody
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 127:5           _async
packages/hive_ce_flutter/src/hive_extensions.dart 10:27                      HiveX.initFlutter
```

**Version**
 - Platform: Web
 - Flutter version: 3.24.3
 - Hive version: 2.5.0+2

**Comment by rayliverified** [comment]
I'm not familiar with the extensions logic used in Hive, otherwise I'd try to contribute a fix.

**Comment by Rexios80** [comment]
Can you test this PR and see if it resolves your issue?
https://github.com/IO-Design-Team/hive_ce/pull/28

**Comment by Rexios80** [comment]
I'm going to go ahead and release that change because it should fix your issue

**Comment by Rexios80** [comment]
Fixed in `hive_flutter:2.1.0`

**Comment by rayliverified** [comment]
Thank you so much @Rexios80, the update fixes the issue!
Hugely appreciate the quick release mate :)
